snap/snapcraft.yaml

```diff
@@ -29,32 +29,22 @@ plugs:
     interface: content
     target: $SNAP_DATA/config/device-rfid-llrp
 
-  # deprecated
-  device-config:
-    interface: content 
-    target: $SNAP_DATA/config/device-rfid-llrp
-
 apps:
   auto-configure:
-    adapter: none
     command: bin/auto-configure.sh
-    command-chain:
-      - snap/command-chain/snapcraft-runner 
```

Comment:
Does the auto configure app still work? It may be missing some environment variables for path definitions.

Reply:
Yes, it still work:
```
$ edgex-device-rfid-llrp.auto-configure  a7910d82-69ae-ea21-214d-fd1326e68545
Dependencies Check: Success
      Consul Check: ...
Failed! curl returned a status code of '7'
* Error code '7' denotes 'Failed to connect to host or proxy'
* Is Consul deployed and accessible?
```
The `snap/command-chain/snapcraft-runner` has been removed by snapcraft because it was not overridable. See ["Snapcraft provided environment" section](https://forum.snapcraft.io/t/micro-howto-migrate-from-core20-to-core22/30188).

Reply:
There are errors so it isn't working. But we can let it go since the errors aren't introduced by this PR. Hopefully we don't have new surprises when the rest of the script gets executed.